class.jetpack-gutenberg.php

```diff
@@ -556,4 +556,32 @@ public static function enqueue_block_editor_assets() {
 
 		wp_enqueue_style( 'jetpack-blocks-editor', $editor_style, array(), $version );
 	}
+
+	/**
+	 * Some blocks do not depend on a specific module,
+	 * and can consequently be loaded outside of the usual modules.
+	 * We will look for such modules in the extensions/ directory.
+	 *
+	 * @since 7.1.0
+	 */
+	public static function load_independent_blocks() {
```

Comment:
There's something funky about term "independent"  🤔 ...that said I don't have anything else to suggest and I'm ok with this. 😄 

I have an incling we'll eventually one day make all our blocks module-independent and re-visit this by renaming this to `load_blocks()`. 😄

Reply:
This may be my ESL at play :)

I wanted to denote that those blocks aren't linked to any module. Happy to change that of course, I just can't think of a better word right now.

Reply:
We actually _do_ have module dependent block(s) in that folder:

https://github.com/Automattic/jetpack/blob/02e53f8ad221e6d65e2d296f760102c5f0b14425/extensions/blocks/tiled-gallery/tiled-gallery.php#L12

I'd be up moving rest of the module-dependent blocks here too and do something similar, e.g.:

https://github.com/Automattic/jetpack/blob/02e53f8ad221e6d65e2d296f760102c5f0b14425/modules/related-posts/jetpack-related-posts.php#L71-L78

(related p9dueE-vJ-p2)

...but that's for another day and PR. :-)

Reply:
You're right. It would make even more sense once we move the blocks themselves in there; it will be better and easier for everyone to have everything in one place.

Once this gets merged, I'll take a look at this.

Reply:
Getting this started in #11325